usersync/cookie.go

```diff
@@ -58,10 +57,7 @@ func ReadCookie(r *http.Request, decoder Decoder, host *config.HostCookie) *Cook
 }
 
 // PrepareCookieForWrite ejects UIDs as long as the cookie is too full
-func (cookie *Cookie) PrepareCookieForWrite(cfg *config.HostCookie, encoder Encoder) (string, error) {
-	uuidKeys := sortUIDs(cookie.uids)
-
-	i := 0
+func (cookie *Cookie) PrepareCookieForWrite(cfg *config.HostCookie, encoder Encoder, ejector Ejector) (string, error) {
 	for len(cookie.uids) > 0 {
 		encodedCookie, err := encoder.Encode(cookie)
 		if err != nil {
```

Comment:
This probably should have been pointed out in a previous PR review but, should we return the `encoder.Encode()` error in line 64? It's probably unlikely that an error happens here, but in that case, en empty error message will be returned in line 178 of `endpoints/setuid.go`.

We could probably add a test that mocks an `encoder` to throw an error so we get test coverage of line 64
![image](https://github.com/prebid/prebid-server/assets/15863840/1045e0b2-4fbd-4563-b2a8-5e0d96ef90f1)